CHANGELOG.md

```diff
@@ -302,7 +606,7 @@ As mentioned a few times in the past, the core team is reviewing the repository
      }
    ```
 
-3. Ensure that you have all the babel dependencies to version `^7.0.0` (you may also need to add `"babel-core": "7.0.0-bridge.0"` as a yarn resolution to ensure retro-compatibility).  The Babel team has released a tool, [babel-upgrade](https://github.com/babel/babel-upgrade), that should help you in this migration.
```

Comment:
probably this is unintended, but can you please reverse this phrase once we are close to stable? 😇

Reply:
Right! unintended change somehow snuck in.